AoLibs.Adapters.iOS/FileStorageProvider.cs

```diff
@@ -117,6 +117,19 @@ public void RemoveFile(string path)
             File.Delete(ResolvePath(path));
         }
 
+        public bool CheckIfFileExist(string path)
+        {
+            path = EnsureFileCreated(path);
```

Comment:
Why creating it if you are just cheking if it exists?

Reply:
Too fast copy paste. I m on it